harness/tests/determined/common/experimental/test_workspace.py

```diff
@@ -71,3 +87,21 @@ def test_workspace_constructor_populates_id_from_name(
 def test_workspace_constructor_doesnt_populate_name_from_id(standard_session: api.Session) -> None:
     ws = workspace.Workspace(session=standard_session, workspace_id=1)
     assert ws.name is None
+
+
+@responses.activate
+def test_list_bounded_resource_pools(
```

Comment:
This function name is hard for me -- I don't know what it's supposed to test. Could you please rename in the style suggested at https://hpe-aiatscale.atlassian.net/wiki/spaces/ENGINEERIN/pages/1351909582/Python+Testing#Best-practices?